index.html

```diff
@@ -11,21 +11,23 @@
       type="module"
     ></script>
   </head>
-  <body class="flex justify-center">
+  <body class="flex justify-center bg-gray-50">
     <div class="flex-grow max-w-2xl mt-10 px-4">
-      <header>
-        <h1 class="font-bold mb-10 text(center gray-600 xl)">
-          Koodikrapula.fi
-        </h1>
+      <header class="flex justify-center">
+        <a href="#">
+          <h1 class="font-bold mb-6 text(center gray-600 2xl) hover:bg-gray-100 hover:text-red-400 w-min p-3 rounded">
```

Comment:
If you want the upper and lower margin to have equal sizes, consider adding `font-mono` for the heading. 🤪

Reply:
Not sure if you meant changing the font but looks good so I changed it 👍🏽